graphql_compiler/schema_transformation/rename_query.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from copy import deepcopy
+
+from graphql.language import ast as ast_types
+from graphql.language.visitor import Visitor, visit
+
+from ..exceptions import GraphQLValidationError
+
+
+def rename_query(ast, renamings):
+    """Translate names of types and root vertex fields using renamings.
+
+    Besides root vertex fields (fields of the query type), no field will be renamed.
+
+    Args:
+        ast: Document, representing a valid query. It is assumed to have passed GraphQL's
+             builtin validation -- validate(schema, ast), in that it has the structure of a
```

Comment:
If you are going to mention a function please be very specific about where this function is located.